src/components/MultiHopTrade/components/TradeSuccess/TradeSuccess.tsx

```diff
@@ -23,16 +29,29 @@ export type TradeSuccessProps = {
   handleBack: () => void
   children: JSX.Element
   titleTranslation?: string | [string, InterpolationOptions]
-  descriptionTranslation?: string | [string, InterpolationOptions]
-}
-
-const pairProps = { showFirst: true }
+} & (
+  | {
+      sellAsset: never
+      buyAsset: never
+      sellAmountCryptoPrecision: never
+      buyAmountCryptoPrecision: never
+    }
```

Comment:
Preferably blocking:

Do we actually need this? `TradeSuccess` is only used once (in `MultiHopTradeConfirm`). 

In what circumstances do we not pass these values?

Reply:
Mmh, I did this because I thought we might use this screen with limit orders, but I might be wrong?

Reply:
Agree we probably shouldn't early abstract this guy, also feels weird having a discriminated union of `AllProperties | None`, i.e not actually discriminating.

@NeOMakinG what about removing the discriminated union for the time being, and if limit order ends up consuming `<TradeSuccess />`, we can always move these properties under some optional `tradeMetadata` of sorts, WDYT?

Reply:
Updated this in [b1a7cbb](https://github.com/shapeshift/web/pull/8240/commits/b1a7cbbefdcd14c05973b2565dd569c641205b9d)